tests/data/regression/model/model.txt

```diff
@@ -5,7 +5,7 @@ num_tree_per_iteration=1
 label_index=0
 max_feature_idx=9
 objective=regression
-feature_names=Column_0 Column_1 Column_2 Column_3 Column_4 Column_5 Column_6 Column_7 Column_8 Column_9
+feature_names=f0 f1 f2 f3 f4 f5 f6 f7 f8 f9
```

Comment:
Is this file used to test inference?

Reply:
This is used for unit testing inferencing scripts. This is a synthetic model trained on synthetic data.